apps/docs/src/app/core/component-docs/quick-view/examples/quick-view-dialog-example.component.html

```diff
@@ -0,0 +1,75 @@
+<ng-template let-dialog let-dialogConfig="dialogConfig" #quickViewDialog>
+    <fd-dialog [dialogConfig]="dialogConfig" [dialogRef]="dialog">
+        <fd-dialog-header>
+            <h1 fd-title>Details</h1>
+        </fd-dialog-header>
+
+        <fd-dialog-body>
+            <fd-quick-view [id]="data.id">
+                <fd-quick-view-subheader>
+                    <fd-avatar [image]="data.subHeader.avatar" size="s"></fd-avatar>
+                    <fd-quick-view-subheader-title>
+                        {{ data.subHeader.title }}
+                    </fd-quick-view-subheader-title>
+                    <fd-quick-view-subheader-subtitle>
+                        {{ data.subHeader.subtitle }}
+                    </fd-quick-view-subheader-subtitle>
+                </fd-quick-view-subheader>
+
+                <fd-quick-view-group *ngFor="let group of data.groups">
+                    <fd-quick-view-group-title>
+                        {{ group.title }}
+                    </fd-quick-view-group-title>
+                    <fd-quick-view-group-item *ngFor="let item of group.items">
+                        <fd-quick-view-group-item-label>
+                            {{ item.label }}
+                        </fd-quick-view-group-item-label>
+                        <fd-quick-view-group-item-content>
+                            <ng-container [ngSwitch]="item.label">
+                                <a *ngSwitchCase="item.label === 'Mobile' || item.label === 'Phone' ? item.label : ''"
+                                   [href]="'tel:' + item.value"
+                                   [innerText]="item.value"
+                                   fd-link
+                                ></a>
+                                <a *ngSwitchCase="'Email'" [href]="'mailto:' + item.value" [innerText]="item.value" fd-link></a>
+                                <div *ngSwitchDefault [innerText]="item.value"></div>
+                            </ng-container>
+                        </fd-quick-view-group-item-content>
+                    </fd-quick-view-group-item>
+                </fd-quick-view-group>
+            </fd-quick-view>
+        </fd-dialog-body>
+
+        <fd-dialog-footer>
+            <!-- TODO: change fd-dialog-footer-button to fd-button-bar after merging PR 4140 -->
```

Comment:
This has been merged, feel free to use new markup :)

Reply:
Nice, thanks :)